pkg/template_validator.go

```diff
@@ -77,3 +82,10 @@ func isValidImageName(name string) error {
 	}
 	return nil
 }
+
+func hasValidTimeout(timeout int64, name string) error {
+	if timeout > 0 {
+		return nil
+	}
+	return fmt.Errorf(errNoTimeout, name)
```

Comment:
Could you please change all calls of `fmt.Errorf` to `errors.Wrapf` instead?

Reply:
`errors.Wrapf` required `err` as the first argument which we do not have in every case.

Reply:
Actually use the correct function. If no formatting is happening, use `errors.Wrap`, if there's no error available to wrap use `errors.New`